app/assets/stylesheets/calagator/forms.scss

```diff
@@ -11,7 +11,9 @@
 
 ############################################################## */
 
-@import 'datepicker';
+/*
+ *= reuqire jquery-ui/datepicker
```

Comment:
I was under the impression that `*= require` syntax should not be used in `.scss` files. Is that not true? I'm thinking of this: https://github.com/rails/sass-rails#important-note